ext/standard/dir.stub.php

```diff
@@ -2,6 +2,80 @@
 
 /** @generate-class-entries */
 
+#ifdef HAVE_GLOB
+#if defined(GLOB_BRACE) && GLOB_BRACE != 0
```

Comment:
nit: could you please put a `()` around the conditions in the `#if`s? It's because gen_stub.php combines multi-level ifs to one, and not having parentheses is a source of bugs when different operators are involved (once I had such a bug). Now, of course there are only `&`s, but IMO it's better to be defensive.

Reply:
That makes sense. Parentheses added.